GameEngine/GameEngine/Engine/AssetManagement/Instance/MaterialInstance.hh

```diff
@@ -14,8 +14,6 @@ namespace AGE
 
 	struct MaterialInstance
 	{
-		//Key<Material> _material_key;
-		//std::map<std::string, Key<Property>> _properties;
```

Comment:
No we ll need a struct because some day I ll do assets instance counter to be able to unload unused assets